modules/virtualization/microvm/netvm.nix

```diff
@@ -28,42 +31,41 @@
         microvm.hypervisor = "qemu";
 
         networking = {
-          enableIPv6 = false;
-          interfaces.ethint0.useDHCP = false;
-          firewall.allowedTCPPorts = [22];
```

Comment:
I think we will still need inbound ssh allowed - at least for `-debug` targets.

Reply:
I did not change any behavior, just reworked the structure so that all common parts are now in the same place.